content/references/proptypes.md

```diff
@@ -78,6 +80,39 @@ export const defaultConfig = {
 
 Merchant edits to color fields are stored in RGBA format.
 
+### date
+
+![Date Picker](proptype-date.png)
+
+```javascript
+date: {
+  label: 'Date',
+  type: ElementPropTypes.date
+}
+
+export const defaultConfig = {
+  date: new Date(2020, 2, 16)
+}
+```
+
+### dateRange
+
+![DateRange Picker](proptype-dateRange.png)
+
+```javascript
+dateRange: {
+  label: 'Date Range',
+  type: ElementPropTypes.dateRange
+}
+
+export const defaultConfig = {
+  dateRange: [
+    new Date(2020, 2, 16), // start date
+    new Date(2020, 2, 26)  // end date
+  ]
+}
```

Comment:
Similarly to above, I wonder if we should add a note suggesting using this as the default:
```js
const today = new Date();
const tomorrow = new Date();
tomorrow.setDate(tomorrow.getDate() + 1);

export const defaultConfig = {
  dateRange: [
    today, // start date
    tomorrow  // end date
  ]
}
```
(I assume it would work that way and not just save the dates at time of release)

Reply:
This is also a good idea. I'll make the change.